grpc-1.66.yaml

```diff
@@ -82,7 +86,7 @@ pipeline:
       	-DgRPC_BUILD_TESTS=OFF
       cmake --build _build
 
-      GRPC_PYTHON_CFLAGS="-std=c++17" \
+      GRPC_PYTHON_CFLAGS="-std=c++17 -specs abseil-cpp.spec" \
```

Comment:
Does GCC_SPEC_FILE as mentioned in the announcement not work?

https://github.com/orgs/wolfi-dev/discussions/33052

Reply:
Oh huh, I'm not sure it's working for me anymore either.

Reply:
I believe `GCC_SPEC_FILE` worked for me when I tested it by pointing it at a different real spec file. I didn't try pointing it at `/dev/null` + and adding other `CFLAGS` though as that document describes. My goal was to avoid diverting from `openssf.spec` as much as possible. This should only filter out the problematic flag, leaving all of the rest of the `openssf` flags in-tact.